README.md

```diff
@@ -55,8 +55,62 @@ To create custom special processes, add them into the `switch` statement in `Cus
 
 You can add custom item or move effects in `CustomApplyItemEffect` and `CustomApplyMoveEffect`.
 
+Please note that custom move effects are currently *not* handled by the *Metronome* move.
+
 #### Compatiblity with existing patches
-This project aims to keep compatibility with existing patches for move, item and special process effects to some degree. Special process effects using the `ExtractSpCode` patch can be reused without problems if they were imported with an ID lower than 100. Compatiblity with the `ExtractMoveCode` has not been thoroughly tested yet and might potentially cause issues with the *Metronome* move.
+
+**Note: ROMs patched with c-of-time currently experience crashes with the `ExtractSpCode` and `ExtractMoveCode` patch.**
```

Comment:
Isn't `ExtractItemCode` the one with a crash, not `ExtractSpCode?` I've seen at a least a few hacks that manage to use `ExtractSpCode` in tandem with CoT.

Reply:
Yup, that's the one I meant, thanks